spec/system/organisations/students/show_spec.rb

```diff
@@ -250,14 +250,38 @@
 
       before { org_admin_user.update!(organisation: organisation) }
 
-      scenario 'org admin cannot see private notes' do
+      scenario "org admin cannot see private notes" do
         sign_in_user org_admin_user,
                      referrer: org_student_path(student_org_admin)
 
         expect(page).to have_text(org_admin_user.name)
-        expect(page).not_to have_text('Notes')
+        expect(page).not_to have_text("Notes")
         expect(page).not_to have_text(coach_note.note)
       end
     end
+
+    context "org admin checks feedback on student submissions" do
+      let(:submission) do
+        create :timeline_event,
+               :evaluated,
+               :passed,
+               :with_owners,
+               latest: true,
+               target: target_l2,
+               owners: [student]
+      end
+      before do
+        4.times do
+          create :startup_feedback,
+                 timeline_event_id: submission.id,
+                 faculty_id: faculty.id
+        end
+      end
+      scenario "org admin can see feedback on student submissions" do
```

Comment:
Readability: Leave gaps between adjacent blocks / paragraphs of code.

```suggestion
      end

      before do
        4.times do
          create :startup_feedback,
                 timeline_event_id: submission.id,
                 faculty_id: faculty.id
        end
      end

      scenario "org admin can see feedback on student submissions" do
```